config/v1alpha1/types_image_policy.go

```diff
@@ -143,6 +151,39 @@ type PolicyFulcioSubject struct {
 	SignedEmail string `json:"signedEmail"`
 }
 
+// PKI defines the root of trust based on Root CA(s) and corresponding intermediate certificates.
+// +openshift:enable:FeatureGate=SigstoreImageVerificationPKI
+type PKI struct {
+	// caRootsData contains base64-encoded data of a certificate bundle PEM file, which contains one or more CA roots in the PEM format.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength=8192
+	CertificateAuthorityRootsData []byte `json:"caRootsData"`
```

Comment:
Why caRootsData as the serialised version instead of certificateAuthorityRootsData?

Reply:
We chose caRootsData to maintain consistency with cosign cli `--ca-roots`, `--ca-intermediates` for the PKI verification, and this follow the existing naming pattern `fucilCAData`